src/api/helpers/map-and-generate-inactive-applications-save-array/index.ts

```diff
@@ -0,0 +1,37 @@
+import { Application } from '.keystone/types'; // eslint-disable-line
+import { APPLICATION } from '../../constants';
+import { InactiveApplicationUpdate } from '../../types';
+
+/**
+ * mapAndGenerateInactiveApplicationSaveArray
+ * generates array of objects for updating the db
+ * object contains where object with id of application to be updated
+ * contains data object with fields which are updated
+ * @param {Array<InactiveApplicationUpdate>} applications: array of application ids and statuses
+ * @returns {Array<InactiveApplicationUpdate>}: Array of objects for updating db
+ */
+const mapAndGenerateInactiveApplicationSaveArray = (applications: Array<Application>) => {
```

Comment:
Not sure about the naming, how about

`mapInactiveApplications`

Could we use a `.map` instead of `.forEach`?

Maybe a `.map` with a `mapInactiveApplication` for each object in the array 🤔

Reply:
done